assets/scss/modules/_datatables.scss

```diff
@@ -104,7 +104,11 @@
   .sorting_desc_disabled {
     background: url("../images/sort_desc_disabled.png") no-repeat scroll center right;
   }
-
+  
+  .sorting--hidden {
```

Comment:
You'll just need to take this out again for Part 3 so why bother putting it in at all?